calvinpete/Glokeeper-backend
# #183110905 configure the static path in settings.py

#### What does this PR do?
This PR configures static path to allow access to the web assets in the settings.py

#### Description of Task to be completed?
- add the static path in `STATISTIC_DIRS` in the settings.py file

#### How should this be manually tested?
- Run `pyhton manage.py makemigrations` to update the database models
- Run `pyhton manage.py migrate` to enforce the changes
- Run the application locally `pyhton manage.py run` to see if any warnings concerning the static path are raised.

#### What are the relevant pivotal tracker stories?
[#183110905](https://www.pivotaltracker.com/story/show/183110905)
